matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/account/ChangePasswordTest.kt

```diff
@@ -43,9 +43,7 @@ class ChangePasswordTest : InstrumentedTest {
         val session = commonTestHelper.createAccount(TestConstants.USER_ALICE, SessionTestParams(withInitialSync = false))
 
         // Change password
-        commonTestHelper.runBlockingTest {
```

Comment:
the `runCryptoTest` & `runSessionTest` are now suspending which means `runBlockingTest` is the equivalent of running in the original scope